common/reactive-grpc-common/src/main/java/com/salesforce/reactivegrpc/common/AbstractSubscriberAndProducer.java

```diff
@@ -459,4 +449,115 @@ private static Throwable prepareError(Throwable throwable) {
         }
     }
 
+    /**
+     * Implementation of Cancelled Queue Subscription which is used as a marker of
+     * cancelled {@link AbstractSubscriberAndProducer} instance.
+     */
+    private static class CancelledQueueSubscription implements Subscription, Queue {
+
+        static final String NOT_SUPPORTED_MESSAGE = "Although CancelledQueueSubscription implements Queue it is" +
```

Comment:
Message not needed since class is `private static`. There is no way for a user to instantiate this class.

Reply:
fixed